### ISSUE TITLE
Unable to handle popup windows in Cypress

### ISSUE THREAD

**Issue by nirav-maru** [cypress-io/cypress#27681]
### What would you like?

Since many Websites integrate with new payment methods like Paypal, Venmo and currently Cypress is not able to automate them as new popup window opens where user login to payment method accounts.

Due to such limitation we are not able to automate end-to-end website for our Clients.

is there any scope of improvement for this feature to be integrate in upcoming releases.

### Why is this needed?

Since Cypress doesn't support to handle new popups like in case of payment method Paypal, its login form opens in new popup & due to Cypress limitation it can't be automated.

Thus from Clients view, their website can't be automated from end-to-end with Cypress.
 
So we are looking ahead from Cypress team to integrate such feature asap in upcoming releases.

### Other

Consider it on high priority

**Comment by bemanuel-trove** [comment]
+1! Please add this feature!

**Comment by gsouf** [comment]
That's not only for paywall. There is a wide a range of scenarios in which cypress would benefit from having the ability to test a new window/popup

**Comment by hbhuiyan-stax** [comment]
any update on this? It's been two yers

**Comment by MDSerialDesignationN** [comment]
It would be really nice to be able to handle popups, especially prompts. Currently, when we create a mail-template on our website there is a popup when you hit save to type in the name for the template. but we can't automate this step because cypress cannot handle it.

**Comment by MDSerialDesignationN** [comment]
> any update on this? It's been two yers

I don't think so. I am struggling on this issue for like 13 Hours and can't get it to work.

**Comment by muz234** [comment]
Playwright and other test frameworks work flawless with handling `pop-up` window box. Cypress is only one with lack of support for this scenarios. I wonder if there is any work around or moving to a different test framework would be better option?
